io.github.johannesboehler2.BmiCalculator.json

```diff
@@ -41,7 +41,7 @@
                     "type" : "git",
                     "url" : "https://github.com/johannesboehler2/bmicalculator.git",
                     "tag" : "v1.0",
-                    "commit": "6af5526acea10914d58768d4b963b6e40d442764"
+                    "commit": "8cc6d4136a4530d45801aa95584059b1e906947a"
```

Comment:
I will repeat one more time. Re-tagging, aka changing a published tag, is a VERY VERY BAD practice.

Reply:
that's because the tag is still there. So it expect the commit to match the tag.

Reply:
which is also why retagging is bad.

Reply:
ok

Reply:
I give up, I am not able to full fill the requirements of Flathub.